doc/reference/modules/configuration.xml

```diff
@@ -1002,12 +1002,6 @@ ISession session = sessions.OpenSession(conn);
   <session-factory>
 
     <!-- properties -->
-    <property name="connection.provider">
-      NHibernate.Connection.DriverConnectionProvider
-      </property>
-    <property name="connection.driver_class">
-      NHibernate.Driver.SqlClientDriver
-    </property>
```

Comment:
Our doc examples incite users to use an obsolete driver for SqlServer: I have purged unneeded parameters.